LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, SciTools-incubator
+Copyright (c) 2020 - 2022, SciTools-incubator
```

Comment:
Not sure that we need to add an end date to this, some projects do and some don't

I'm not a lawyer, but don't see the harm :+1:

Reply:
I've no strong feelings either way. Totally happy to leave it as you've changed it.

Perhaps raise it as a post discussion with other core devs? That seems reasonable to me :+1: